utils/update_checkout/update-checkout-config.json

```diff
@@ -113,6 +113,9 @@
         },
         "zlib": {
           "remote": { "id": "madler/zlib" }
+        },
+        "mimalloc": {
+          "remote": { "id": "microsoft/mimalloc" }
```

Comment:
Can you add
```suggestion
          "remote": { "id": "microsoft/mimalloc" },
          "platforms": [ "Windows" ]
```

This way we're only downloading it on Windows.

Reply:
I think that is best done in a follow up if we want to remove some of these repositories from being checked out (e.g. playground-support).

Reply:
I'm fine if there's a follow-up for the others, but this is new stuff.

Reply:
Doing that as a single pass is better IMO.